mapping/sustainability.yaml

```
@@ -41,7 +40,7 @@
     module: Economic and fiscal
     indicator: Economic viability
     disclosure format: Disclose the methodology used to calculate the life-cycle cost. The methodology ought to specify whether income and externalities are included in the calculation and the common date, discount rate and period of analysis used.
-    mapping: 'Project Level: [Add a project document](../common.md#add-a-project-document) and set `.documentType` to ''lifeCycleCostMethodology''.'
+    mapping: 'Project Level: [Add a project document](../common.md#add-a-project-document) and set its [`.documentType`](project-schema.json,/definitions/Document,documentType) to ''lifeCycleCostMethodology''.'
```

```suggestion
    mapping: |
        Project Level: 
        
        [Add a project document](../common.md#add-a-project-document) and set its [`.documentType`](project-schema.json,/definitions/Document,documentType) to 'lifeCycleCostMethodology'.
```
Also I can't select it for a suggestion as it's not a change from this PR but noticed that the `documentType` is "lifecycleCostMethodology" in the accompanying example, but it should be "lifeCycleCostMethodology"

---

Fixed in 73bc7abfdb6a4bf562298f1f80ac2d6cfd7656ec